web/src/components/feature/package_dialog/FluidDialog.tsx

```diff
@@ -102,6 +112,17 @@ export const FluidDialog = ({
               multiline
               rows={6}
             />
+            <Autocomplete
+              label="Template"
+              options={packages}
+              optionLabel={(option) => option.name}
+              onOptionsChange={(changes) => {
+                setComponentRatios(changes.selectedItems[0].components)
+                setSelectedPackage(changes.selectedItems[0])
+              }}
+              selectedOptions={selectedPackage ? [selectedPackage] : []}
+              autoWidth
+            />
```

Comment:
We don't need this component to be controlled, so I suggest simplifying it
```suggestion
            <Autocomplete
              label="Template"
              options={packages}
              optionLabel={(option) => option.name}
              onOptionsChange={(changes) => {
                setComponentRatios(changes.selectedItems[0].components)
              }}
              autoWidth
            />
```
and then you can remove the selectedPackage state

Reply:
good point, fixed